### DISCUSSION TITLE
migrating-comments-from-dsqus-to-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#376] [Announcements]
# migrating-comments-from-dsqus-to-giscus/

In this post I describe a .NET script/tool I created to migrate the comments on my blog posts from my legacy Disqus account to GitHub discussions

https://andrewlock.net/migrating-comments-from-dsqus-to-giscus/

**Comment by dvdstelt** [comment]
This info is extremely helpful, including the repo and the next article about Octokit GraphQL
I'm converting items from my Wordpress blog to Disgus.

One thing that was missing though, is that many people did `@dennis` or someone else's name when replying. I just converted a whole lot of comments and suddenly get remarks from people that get spammed by my discussions. With you, afaik, it was more the conversion from Disqus to Github accounts, with me it's random @ mentions :-)

Anyway, thanks for the explanation and code, extremely helpful.

**Reply by InvictusNavarchus** [reply]
That's what I thought as well. The @dvdstelt is quite handy. It's unfortunate there is no auto-completion for it.

**Comment by dracan** [comment]
Andrew - you're an absolute star! ⭐
I've recently switched my old site from a custom build to instead use [11ty](https://www.11ty.dev/), and at the same time, wanted to move away from Disqus because of their ads. I ported them all this morning. Your posts and GitHub project have made this WAY easier than it otherwise would have been. Thank you so much for your time and effort doing this! 🙏

**Reply by andrewlock** [reply]
🎉Really glad to hear it! I wanted to move for the same reason and I have to say it was a great move, so I'm sure you'll be happy with it 👍 And really glad to hear it all made it easier for you - that's the goal after all 😀
